pkg/listening-sockets/listening-sockets.go

```diff
@@ -263,7 +263,7 @@ func filterEntries(ssEntries []string) []string {
 func parseComDetail(ssEntry string) *types.ComDetails {
 	serviceName, err := extractServiceName(ssEntry)
 	if err != nil {
-		log.Debugf(err.Error())
+		log.Debug(err.Error())
```

Comment:
Is this related to this change?

Reply:
no we talked about this that i had version of linter more than the old one so it make this failing on linting

Reply:
i make it in another commit

Reply:
This change is in two commits, please check